main.go

```diff
@@ -121,10 +121,6 @@ func main() {
 		panic(errors.New("gha-find-replace: expected with.find to be a string"))
 	}
 
-	if replaceErr != nil {
-		panic(errors.New("gha-find-replace: expected with.replace to be a string"))
```

Comment:
I think we shouldn't be deleting this check and instead assign `replace = ""` instead of panicing.

Reply:
Sure, works for me :) changed.